skbeam/io/fit2d_mask.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import print_function, division
+
+import os
+
+import numpy as np
+import math
+
+from skbeam.io import logger
+
+
+def fit2d_save(mask, filename, dir_path=None):
+    """
+    Compresses and wraps the mask for Fit2D use
+    Parameters
+    ----------
+    mask: ndarray
+        The mask
+    filename: str
+        The filename
+    dir_path: str, optional
+        Path to the destination file
+    """
+    # Fit2d works with inverted masks
+    mask = np.flipud(mask)
+    mask = np.abs(mask - 1)
+    b1, b2 = mask.shape
+    b28 = b2 // 8
+    c = np.zeros((b1, b28), dtype=np.uint8)
+    print('start compression')
+    # bitmap compression for Fit2D io
+    # TODO: possibly parallizable
+    for i in range(0, b1):
+        j = 0
+        while j < b2:
+            c[i, math.floor(j / 8)] = mask[i, j]
+            for k in range(1, 8):
+                c[i, math.floor(j / 8)] += mask[i, j + k] * (2 ** k)
+            j += 8
+    maskname = _create_file_path(dir_path, filename, '.msk')
+    with open(maskname, "wb") as fout:
+        c.tofile(fout, "", "%x")
+    print('mask compressed')
+    # Giant long mask header, must be there for fit2d to work with the mask
+    header = b"M\x00\x00\x00A\x00\x00\x00S\x00\x00\x00K\x00\x00\x00\x00\x08" \
```

Comment:
Could you make this a module level variable so we dot have to create one every time this function gets called?